drivers/ovsdriver.go

```diff
@@ -204,6 +209,23 @@ func (d *OvsDriver) DeleteNetwork(id, encap string, pktTag, extPktTag int, gatew
 		sw = d.switchDb["vlan"]
 	}
 
+	// Delete infra nw endpoint if present
+	if nwType == "infra" {
+		hostName, _ := os.Hostname()
```

Comment:
Both create and delete endpoint is explicitly triggered by the mgmt plugins - they trigger endpoint create/delete on both master and local-netplugin. Similarly infra-nw create/delete triggers endpoint create/delete on both master and local-netplugin.
When non-master nodes receive infra network delete, network is already deleted on master nodes. Endpoint delete requires network-specific information (encap and nwtype), so we will not have sufficient context to clean the endpoint after network delete processing is complete on a node.